client/src/AppBundle/Resources/views/Report/Document/step1.html.twig

```diff
@@ -18,16 +20,25 @@
 
     {{ form_start(form, {attr: {novalidate: 'novalidate' }}) }}
 
+    <h2 class="heading-medium">{{ (page ~ '.step1Heading') | trans }}</h2>
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                {{ (page ~ '.selectHint1') | trans }}
```

Comment:
Ol' fussport Tyler
```suggestion
                <p>{{ (page ~ '.selectHint1') | trans }}</p>
```

Reply:
Sooo this was actually a conscious design decision to match the prototype as it looked like there was no spacing between hint1 and hint2:

<img width="550" alt="Screenshot 2020-03-26 at 11 05 44" src="https://user-images.githubusercontent.com/47983793/77640171-d4b6b800-6f51-11ea-8eac-9b885d06d2f9.png">

Is there a more semantic way to lay this out?

Reply:
I see, yep, probably just this then:
```
             <p>
                 <strong>{{ (page ~ '.selectHint1') | trans }}</strong>
                 <br>
                 {{ (page ~ '.selectHint2') | trans }}
             </p>
```

Reply:
Makes sense 👍 to confuse matters, I provided the prototype for step2 as I couldn't see a prototype for step1. Instructions on the ticket was to add the same inset message but with the first line as regular text. I'll add the `<br>` to both.